src/components/DesignTheme/manage/Widgets/CharCounterDescriptionWidget.jsx

```diff
@@ -131,7 +131,6 @@ const CharCounterDescriptionWidget = ({
               </div>
             )}
             <TextArea
-              maxLength={160}
```

Comment:
Questo lo lascirei comunque per mantenere il limite di inserimento e non permettere all'utente di inserire un testo oltre i 160 caratter

Reply:
Questa PR è appunto per rimuovere il limite blindato, perchè dalle migrazioni arriverranno descrizioni più lunghe.
In questo modo rimane come suggerimento e non come limite imposto.

Reply:
ahh ok capito, allora va bene